schemas/input_json_format.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "The root schema",
+  "description": "https://github.com/vowpalwabbit/vowpal_wabbit/wiki/JSON",
+  "default": {},
+  "examples": [],
+  "required": [],
+  "definitions": {
+  },
+  "additionalProperties": true,
+  "patternProperties": {
+    "^[^_].*": {
+      "type": [
+        "string",
+        "number",
+        "boolean",
+        "array",
+        "object"
+      ],
+      "items": {
+        "type": "number"
+      }
+    },
+    "^_": {
```

Comment:
Do we allow a prop which is just `_`?

Reply:
Yes, via "Top-level properties starting with _ are ignored" but it is not that clear because it's using regex so as long as it matches it will return True. Maybe a more explicit check would be `^_.*`

Reply:
Wouldn't it be more accurate to say all top level properties are ignored except for the ones we add a specific schema for?

Reply:
That doesn't help because: "Top-level properties are considered features for the default namespace.
Top-level properties of type object or array are considered namespaces.". This just addresses the option for users to use names starting with _ to be ignored as namespaces and features.

Reply:
Wait, depends on our requirements. I interpreted that feature as a way to have a flexible format that allowed that but not sure if that was the intent.

Reply:
I'd need to check but I hope the json format ignores all "_.*" properties that are not specific ones at the root level.

Reply:
Well, even if it is ignored it should be validated at the json level. i.e. if we drop this line the validator will fail if someone makes use of that ignored property feature.